index.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const movies = require("./movies");
+const app = express();
+const {
+  getAllMovies,
+  getMovieByDirector,
+  getMovieByTitle,
+  saveNewMovie,
+} = require("./controllers/movies");
+
+app.get("/movies", getAllMovies);
+app.get("/movies/:directors", getMovieByDirector);
+app.get("/movies/:title", getMovieByTitle);
```

Comment:
The only functionality of your submission that doesn't work right away is searching for movies by title, however searching by director works perfectly fine. I was able to switch this by commenting out line 13, as it would seem that the ordering of these app.get statements forces all queries of '/movies/search' to solely search by directors, blocking the possibility of searching by movie title. A more sure-fire remedy to this may be to combine the functionality of lines 13 and 14 into one 'search' option which checks for a user-input query of either directors or title.